paimon-core/src/main/java/org/apache/paimon/table/system/ManifestsTable.java

```diff
@@ -200,6 +200,8 @@ private static List<ManifestFileMeta> allManifests(FileStoreTable dataTable) {
         SnapshotManager snapshotManager = dataTable.snapshotManager();
         Long snapshotId = coreOptions.scanSnapshotId();
         String tagName = coreOptions.scanTagName();
+        Long timestampMills = coreOptions.scanTimestampMills();
+
         Snapshot snapshot = null;
```

Comment:
Variable 'snapshot' initializer 'null' is redundant.
Change "Snapshot snapshot = null;" to "Snapshot snapshot;".